src/components/SingleArticle.jsx

```diff
@@ -31,7 +52,7 @@ export default function SingleArticle() {
                 <p>{article.body}</p>
                 <p>Topic: {article.topic}<br/> 
                     Author: {article.author}<br/> 
-                    Date Published: {article.created_at}
+                    Date Published: {article.created_at? article.created_at.split('T')[0]: null}
```

Comment:
I can also add as many votes as I want, should it be limited to one per user per article ?